sys/include/shell.h

```diff
@@ -21,9 +21,10 @@
 #define SHELL_H
 
 #include <stdint.h>
-#include "periph/pm.h"
 
 #include "kernel_defines.h"
+#include "periph/pm.h"
```

Comment:
Seems like an unrelated style change. Maybe split this out into its own commit?

Reply:
hm I'm lazy and would rather save the reset/commit/commit cycle.

Reply:
Make a fixup commit this undoes this, than create a revert commit of the fixup. Then keep the reverting commit and squash the fixup in.

It would be nice to have a vim plugin to automate this.

Reply:
ok, style fix reverted